core/server/api/themes.js

```diff
@@ -26,11 +27,24 @@ var debug = require('debug')('ghost:api:themes'),
 themes = {
     browse: function browse() {
         debug('browsing');
-        var result = packageUtils.filterPackages(themeList.getAll());
+        var result = themeList.toAPI(themeList.getAll(), settingsCache.get('activeTheme'));
```

Comment:
two small thoughts on this for later:
- the themeList has access to `getAll`
- it's a bit weird that we have to push the active theme as parameter somehow - shouldn't the theme list already know which one is active?